source.mak

```diff
@@ -430,6 +430,42 @@ WIN32_HEADS = main/e_msoft.h
 WIN32_SRCS = win32/mkstemp/mkstemp.c
 WIN32_OBJS = $(WIN32_SRCS:.c=.$(OBJEXT))
 
+# common to MVC and MINGW
+CMN_GNULIB_HEADS = \
```

Comment:
Could you use `COMMON_` as a prefix instead of `CMN_`?

Reply:
Sure.